### PR TITLE
Update sort function

### PR DESCRIPTION
**Pull request by haslinghuis**
Use `JQuery` instead of `QuerySelector`

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by sugaarK — APPROVED** [review]

File: src/js/utils/common.js [hunk 1]
```diff
@@ -87,20 +87,29 @@ export function getTextWidth(text) {
...
+
+    op.sort((a, b) => {
+        console.log(a.value, a.text);
+        if (a.text === text) {
+            return -1;
+        }
+        if (b.text === text) {
+            return 1;
+        }
+        return a.text.localeCompare(b.text, window.navigator.language, { ignorePunctiaton: true });
```
Comment (McGiverGim) [inline src/js/utils/common.js:108]: It seems the `localeCompare` does not complain about typos :)

```suggestion
        return a.text.localeCompare(b.text, window.navigator.language, { ignorePunctuation: true });
```

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: src/js/utils/common.js (hunk 1 above)
Comment (HThuren) [reply src/js/utils/common.js:108]: ... and it seems to work, strange ...

Comment (McGiverGim) [reply src/js/utils/common.js:108]: I suppose simply ignores the parameter, and not ignores de punctuation.

Comment (haslinghuis) [reply src/js/utils/common.js:108]: https://reference.wolfram.com/language/ref/IgnorePunctuation.html :)

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**